Doc/whatsnew/3.13.rst

```diff
@@ -166,6 +166,11 @@ opcode
 os
 --
 
+* :func:`os.cpu_count` and :func:`os.process_cpu_count` can be overrided through
+  the new environment variable :envvar:`PYTHONCPUCOUNT` or the new command-line option
+  :option:`-X cpu_count <-X>`. This option is useful for users who need to limit
+  CPU resources of a container system.(Contributed by Donghee Na in :gh:`109595`)
```

Comment:
Document it *after* the item below which documents the addition of process_cpu_count().